### PR TITLE
Eric tests 1

### PR DESCRIPTION
**Pull request by edarsey**
This is the first set of tests for the CRUD on the posts model. I don't believe the model has been created yet. I only want to add the file "spec/models/user_CRUD_posts_spec.rb" to the app

### CODE REVIEW

File: spec/models/user_spec.rb [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+    #user successfully deletes item
+    #user is redirected to index page that no longer contains item title
+    #later we can write this test to perform a search query, that if passing, will no longer find the title.
+  end
+
+  pending "User who is not the creator of the item tries to delete it" do
+    fill_in "Email" with:
+    fill_in "Password" with:
+    fill_in "Password Confirmation" with:
+    #user that created item is different than logged in user!
```
Comment (dreamingfurther) [inline spec/models/user_spec.rb]: its not good practice to commit "commented code" to master

File: spec/models/user_spec.rb [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+    visit '/'
+    click_link('Ylse: A Web Series')
+    click_button('Delete')
+
+    expect(page).to_not have_content('Ylse: A Web Series')
+
+
+
+
+    # fill_in "Email" with:
```
Comment (dreamingfurther) [inline spec/models/user_spec.rb]: its not good practice to commit "commented code" to master

File: spec/models/user_spec.rb [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+
+  scenario "User adds item unsuccessfully" do
+    visit '/'
+    click_button('Post a Series')
+    fill_in('Media Type') with: title_one.media_type
+    fill_in('Description') with: title_one.description
+    fill_in('Link') with: title_one.link
+
+    expect(page).to_not have('http://www.eastwillyb.com/')
+    expect(page).to_not have('Willie Jr.’s got his work cut out for him. Running Bushwick’s favorite Latino bar isn’t easy when the old timers retire, the new generation forgets their culture, and the hipsters start taking over the hood. As if it couldn’t get worse, Maggie, Willie Jr.’s ex-fiance, has taken up with Willie Jr.’s arch-nemesis, Albert, and is skillfully transforming Albert’s competing neighborhood bar into a moneymaker, as they cater to the whims of the new residents. Along with the help of his bar regulars, Willie Jr. tries to come up with a solution quickly, or risk losing the two loves of his life: Maggie and his sports bar.')
```
Comment (dreamingfurther) [inline spec/models/user_spec.rb]: what's the story with these "huge" strings in the test files? Is it "necessary" to have these strings be this big? they make it hard to really read these tests well/easily.

File: .hound.yml [hunk 4]
```diff
@@ -119,7 +119,7 @@ LineEndConcatenation:
   Enabled: false
 
 LineLength:
-  Max: 80
+  Max: 200
```
Comment (edarsey) [inline .hound.yml]: This is to cover the old HoundCI errors that we agreed were not significant.

File: spec/models/user_CRUD_posts_spec.rb [hunk 5]
```diff
@@ -0,0 +1,109 @@
+require 'rails_helper'
+before each do
```
Comment (CeeNoriega) [inline spec/models/user_CRUD_posts_spec.rb]: this should be before(:each) do

File: spec/models/user_CRUD_posts_spec.rb [hunk 6]
```diff
@@ -0,0 +1,109 @@
...
+
+  scenario "User is the creator of the artwork and successfully deletes the artwork" do
+    visit '/'
+    click_link('Ylse: A Web Series')
+    click_button('Delete')
+
+    expect(page).to_not have_content('Ylse: A Web Series')
+  end
+
+  pending "User who is not the creator of the item tries to delete it" do
```
Comment (CeeNoriega) [inline spec/models/user_CRUD_posts_spec.rb]: remove unused code-- you don't want this in your master branch

**Pull request merged**